tasks/install-matlab/v0/task.json

```diff
@@ -19,6 +19,7 @@
         }
     },
     "messages": {
+        "InstallNotSupportedOnSelfHosted": "The install task is not supported on self-hosted agents. Use the standard MATLAB installer to install MATLAB on this system.",
```

Comment:
We don't start our error messages with an article. Also to conform to the README, I would say "Install MATLAB task". Here is what I propose:

Install MATLAB task is not supported on self-hosted agents. To install MATLAB on this machine, use the standard installer.

Reply:
Updated to reflect your proposal